website/content/en/status/report-2024-04-2024-06/erlang.adoc

```diff
@@ -0,0 +1,37 @@
+=== FreeBSD Erlang Ecosystem Ports update
+
+Links: +
+link:https://wiki.freebsd.org/Erlang[FreeBSD Erlang wiki] URL: link:https://wiki.freebsd.org/Erlang[https://wiki.freebsd.org/Erlang] +
+link:https://erlang.org/[Erlang/OTP language] URL: link:https://erlang.org/[https://erlang.org/] +
+link:https://elixir-lang.org/[Elixir language] URL: link:https://elixir-lang.org/[https://elixir-lang.org/] +
+link:https://gleam.run/[Gleam language] URL: link:https://gleam.run/[https://gleam.run/] +
```

Comment:
For future reference: you don't need to repeat the target URL within [] when it's also the link text. Thus, instead of `link:https://wiki.freebsd.org/Erlang[https://wiki.freebsd.org/Erlang]`, you can write `link:https://wiki.freebsd.org/Erlang[]` and likewise for the other 3.